### PR TITLE
Feat/kill session endpoint

### PR DESCRIPTION
**Pull request by Adriansillo**
* new trackingId parameter to set a custom session id
* new kill endpoint
  * /kill/all?token=TOKEN 
  * /kill/BROWSER_ID?token=TOKEN 
  * /kill/TRACKING_ID?token=TOKEN

### CODE REVIEW

File: src/browsers/index.ts [hunk 1]
```diff
@@ -328,6 +325,23 @@ export class BrowserManager {
     }
   }
 
+  public async killSessions(target: string): Promise<void> {
```
Comment (joelgriffith) [inline src/browsers/index.ts:328]: We should probably throw a 404 or something if the session isn't found. This way folks will know if it was successful or not

File: src/utils.ts [hunk 2]
```diff
@@ -743,23 +743,14 @@ export const parseStringParam = (
...
   defaultValue: string,
-) => {
+): string => {
   const value = params.get(name);
 
   if (value === null) {
     return defaultValue;
   }
 
-  // ?param format (no specified value)
```
Comment (joelgriffith) [inline src/utils.ts:753]: Any reason why this was dropped?

**Review by joelgriffith — CHANGES_REQUESTED** [review]
I think we should 404 if a kill isn't found, great to see those tests! Also a comment on the util changes

File: src/utils.ts (hunk 2 above)
Comment (Adriansillo) [reply src/utils.ts:753]: I understand parseStringParam method must return an string, similar to parseBooleanParam and parseNumberParam which return a boolean and a number.
it was returning a number for "123" input and could return an object for a valid json object. 
Not sure if my interpretation is correct.

Comment (joelgriffith) [reply src/utils.ts:753]: Yup, that interpretation is correct and I agree with the changes after looking. Thanks!

**Review by joelgriffith — APPROVED** [review]

**Pull request merged**